x/twap/api.go

```diff
@@ -58,17 +93,16 @@ func (k Keeper) GetArithmeticTwap(
 	if err != nil {
 		return sdk.Dec{}, err
 	}
-	return computeArithmeticTwap(startRecord, endRecord, quoteAssetDenom)
+	return strategy.computeTwap(startRecord, endRecord, quoteAssetDenom)
```

Comment:
doesnt this change to `getTwapToNow` anyways if `endTime.Equal(ctx.BlockTime())`?

Reply:
oops this change wasn't supposed to be in this PR, but endTime = BlockTime logic is covered in line 84. We can follow up this conversation in this PR https://github.com/osmosis-labs/osmosis/pull/3529